packages/eslint-plugin/tests/rules/strict-boolean-expressions.test.ts

```diff
@@ -1643,6 +1643,7 @@ if (x) {
         !obj
         obj || 0
         obj && 1 || 0
+        obj ?? 1
```

Comment:
[Testing] This case doesn't seem to be reported at all (which is great!), but it is currently placed in the `invalid` section of the rule tests. I think it would be better to put it in the `valid` section. What do you think?